app/components/Pages/AuthenticationPageContainer/AuthenticationPageContainer.stories.js

```diff
@@ -0,0 +1,24 @@
+import AuthenticationPageContainer from '.';
+
+export default {
+  title: 'Pages/AuthenticationPageContainer',
+  component: AuthenticationPageContainer,
+};
+
+const Template = (...args) => <AuthenticationPageContainer {...args} />;
+
+export const DesktopView = Template.bind({});
+
```

Comment:
don't you think you could use here `defaultViewport: 'responsive'`
?
like this:

```suggestion
export const DesktopView = Template.bind({});
DesktopView.parameters = {
  viewport: {
    defaultViewport: 'responsive',
  },
};
```